src/main.rs

```diff
@@ -99,6 +103,7 @@ async fn main() -> Result<(), Error> {
                     _ = timer.tick() => {
                         let _ = maintenance_store.vacuum_db().await;
                         info!("ran maintenance vacuum");
+                        let _ = runtime_config.reload_config().await;
```

Comment:
Could we update the log message to indicate that config is reloaded as well? It would be good if we could return a `Result` here that indicates if the config has changed. If we had that we could clean out killed tasks only when configuration is changed.

Reply:
Updated `reload_config` to log when runtime config has changed and successfully reloaded. This also returns a result indicating whether the config has been reloaded.